code/modules/shuttle/special.dm

```diff
@@ -162,10 +166,20 @@
 	unique_name = FALSE // disables the (123) number suffix
 	initial_language_holder = /datum/language_holder/universal
 	default_storage = null
+	///The roundend callback tied to the drone that we use to remove its pacification.
+	var/datum/callback/roundend_callback
+
+/mob/living/basic/drone/snowflake/bardrone/Destroy()
+	LAZYREMOVE(SSticker.round_end_events, roundend_callback)
+	roundend_callback = null
+	return ..()
 
 /mob/living/basic/drone/snowflake/bardrone/Initialize(mapload)
 	. = ..()
 	AddComponentFrom(ROUNDSTART_TRAIT, /datum/component/area_based_godmode, area_type = /area/shuttle/escape, allow_area_subtypes = TRUE)
+	ADD_TRAIT(src, TRAIT_PACIFISM, INNATE_TRAIT) //drone is pacified
+	roundend_callback = CALLBACK(src, PROC_REF(round_end_depacify))
```

Comment:
```suggestion
	roundend_callback = TRAIT_CALLBACK_REMOVE(src, TRAIT_PACIFISM, INNATE_TRAIT)
```
This would be how to do it if that's what you want.